### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by peekxc** [RUrlus/carma#79]
While compiling carma from source and installing into my system library works fine, upon trying to include the interface target in a header file I receive this error: 
```cpp
In file included from ../src/tallem/carma_svd.cpp:2:
/usr/local/carma/include/carma:18:12: fatal error: 'cnalloc.hpp' file not found
  #include <cnalloc.hpp>
           ^~~~~~~~~~~~~
```

Upon inspection, the interface target file `carma` looks like:

```cpp
#ifndef INCLUDE_CARMA_H_
#define INCLUDE_CARMA_H_
...
#ifndef CARMA_ARMA_ALIEN_MEM_FUNCTIONS_SET
  #if defined(ARMA_VERSION_MAJOR)
    #error "carma: please include the armadillo header after the carma header or use the CARMA CMake build"
  #endif
  #include <cnalloc.hpp>
#endif
...
#include "carma_bits/nparray.h"
#include "carma_bits/arraystore.h"
#include "carma_bits/converters.h"
#endif  // INCLUDE_CARMA_H_
```
Shouldn't the include with `cnalloc.hpp` use the relative-path include pattern, like the headers at the bottom? Or is this a build configuration problem?

**Comment by conradsnicta** [comment]
@RUrlus This indeed does look like it should be `#include "carma_bits/cnalloc.hpp"` instead of `#include <cnalloc.hpp>`

Appears to have been introduced with commit https://github.com/RUrlus/carma/commit/62ac5ffb6c1637b7861570d0495b17ba061a3ccc

**Comment by RUrlus** [comment]
@peekxc Thanks for reporting. You and @conradsnicta are right, I guess this slipped through CI/CD as I never test an installed version.

**Comment by RUrlus** [comment]
@peekxc Fixed in release 9ebd5f4832f196032aa73dbf14908d9224e5a4ee. @conradsnicta Thanks for the quick PR
